bin/benchmark/main.c

```diff
@@ -55,6 +55,10 @@ static void s_runcrc64(struct aws_byte_cursor checksum_this) {
     (void)crc;
 }
 
+#define KB_TO_BYTES(kb) ((kb) * 1024)
+#define MB_TO_BYTES(mb) ((mb) * 1024 * 1024)
+#define GB_TO_BYTES(gb) ((gb) * 1024 * 1024 * 1024ULL)
```

Comment:
utterly trivial: why does GB have `ULL` in it but the others don't? they're all passed to the same `size_t buffer_sizes[] = {...}`  later?

Reply:
this is just to work around overflows. not really necessary in this case. i copied those defines from c-s3